napari/layers/shapes/shape_models/shape.py

```diff
@@ -236,6 +237,7 @@ def face_color(self, face_color):
         else:
             rgb = tuple([int(255 * x) for x in self._face_color.rgba[:3]])
             self._face_color_name = '#%02x%02x%02x' % rgb
+        self._face_color = self._face_color.rgba
```

Comment:
can you revert this change - i think it is causing problems with the colors in the shapes layer right now, and we'll have a new PR after this just focused on shapes